projects/plugins/jetpack/extensions/plugins/publicize/components/form/index.js

```diff
@@ -1,69 +1,55 @@
 /**
- * Higher Order Publicize sharing form composition.
+ * Publicize sharing form component.
  *
- * Uses Gutenberg data API to dispatch publicize form data to
- * editor post data in format to match 'publicize' field schema.
+ * Displays text area and connection list to allow user
+ * to select connections to share to and write a custom
+ * sharing message.
  */
 
-/**
- * External dependencies
- */
-import { compose } from '@wordpress/compose';
-import { withSelect, withDispatch } from '@wordpress/data';
-
 /**
  * Internal dependencies
  */
-import PublicizeFormUnwrapped from '../form-unwrapped';
+import PublicizeConnection from '../connection';
+import PublicizeSettingsButton from '../settings-button';
+import MessageBoxControl from '../message-box-control';
+import useSocialMediaConnections from '../../hooks/use-social-media-connections';
+import useSocialMediaMessage from '../../hooks/use-social-media-message';
+
+export default function PublicizeForm( { refreshCallback } ) {
+	const { connections } = useSocialMediaConnections();
+	const { message, updateMessage, maxLength } = useSocialMediaMessage();
+	const { toggleById } = useSocialMediaConnections();
```

Comment:
Can we combine this with `connections`?
```suggestion
	const { connections, toggleById } = useSocialMediaConnections();
```

Reply:
I thought I did 🤔. Checking...

Reply:
addressed here https://github.com/Automattic/jetpack/pull/21090/commits/7325bbe2581f2fa170f9522b72f75ab8c5a73c13. Thanks 👍